plugins/capistrano/content.yaml

```diff
@@ -0,0 +1,47 @@
+title: Capistrano
+author: glaszig
+tags: [ capistrano, deploy, ssh, ruby ]
+logo: capistrano.png
+repo: https://github.com/glaszig/drone-capistrano
+image: https://hub.docker.com/r/glaszig/drone-capistrano
+license: MIT License
+readme: https://github.com/glaszig/drone-capistrano/blob/master/README.md
+description: >-
+  The Capistrano plugin deploys applications via the Capistrano ruby gem.
+example: |
+  kind: pipeline
+  name: default
+
+  steps:
+  - name: deploy production
+    image: glaszig/drone-capistrano:2.7.4
+    settings:
+      tasks: production deploy deploy:restart
+    environment:
+      BUNDLE_PATH: vendor/bundle
+      CAPISTRANO_PRIVATE_KEY:
+        from_secret: production_private_key
+      CAPISTRANO_PUBLIC_KEY:
+        from_secret: production_public_key
```

Comment:
can these go under settings instead of environment?

Reply:
yes.